src/services/rfqm_service.ts

```diff
@@ -698,6 +698,7 @@ export class RfqmService {
             // Terminate with an error transition
             await this._dbUtils.updateRfqmJobAsync(orderHash, {
                 status: RfqmJobStatus.FailedLastLookDeclined,
+                calldata: '', // clear out calldata so transaction can never be submitted, even by accident
```

Comment:
if we are clearing calldata here, do we need to handle some situation where a job is picked up by the worker and the calldata for that order was blanked out?

Reply:
ah - yes, so you're referring to a strange retry scenario. Because the status and the calldata get updated in the same query, I don't think there's a huge risk to entering some illegal state.

However, that said, I believe Kroeger's "repair" mode will fix even those types of issues.

Reply:
That makes sense, thanks!